src/index.html

```diff
@@ -3,12 +3,59 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+  <main>
+    <header class="header">
+      <a href="#home" class="logo" >
+        <img
+          src="logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+
+          <li class="nav_item">
+            <a class="nav_link is-active" href="apple">apple</a>
+          </li>
+
+          <li class="nav_item">
+            <a class="nav_link" href="samsung">samsung</a>
+          </li>
+
+          <li class="nav_item">
+            <a class="nav_link" href="smartphones">smartphones</a>
+          </li>
+
+          <li class="nav_item">
+            <a class="nav_link" href="laptops&computers"
+            data-qa="hover">laptops & computers</a>
```

Comment:
```suggestion
            <a 
              class="nav_link"
              href="laptops&computers"
              data-qa="hover"
            >
              laptops & computers
            </a>
```
check your code style